geozero/src/mvt/tile_value.rs

```diff
@@ -0,0 +1,92 @@
+use crate::mvt::tile::Value;
+use std::hash::Hash;
+
+/// A wrapper for the MVT value types.
+#[derive(Debug, Clone, PartialEq)]
+pub enum TileValue {
+    Str(String),
```

Comment:
I know this is already merged, but it wasn't open for long. I like this tag builder change - it cleans things up a bit. 

One nitpick: I feel like `TileValue::String` would be a better name, so as not to confuse it with rust's `str` type. Is there some reason to call it `Str`, or is just that it's shorter to type?

Reply:
Thx for feedback @michaelkirk! I actually initially called it `String` too, but the issue is that it is identical to the system string, making it impossible to use it without the enum qualifier like here:

https://github.com/georust/geozero/blob/1a1585a9fb9cfbcb3cc8d37280ab8b68746b739c/geozero/src/mvt/tag_builder.rs#L63-L66

So I opted to call it by a different name. I think I have seen this pattern elsewhere in the past, where a string wrapper is called `Str` with a different case to differentiate it while still indicate its close associativity with the string type.

Reply:
Fair point! I guess it's a trade off / matter of taste.